apps/ledger-live-mobile/ios/Podfile

```diff
@@ -73,6 +73,9 @@ target 'ledgerlivemobile' do
           config.build_settings["USE_HEADERMAP"] = "NO"
         end
       end
+      target.build_configurations.each do |config|
+        config.build_settings['CODE_SIGNING_ALLOWED'] = 'NO'
+      end
```

Comment:
Maybe add a comment on why we are doing this since it's not directly related to the PR?

Reply:
reverted the change on @valpinkman 's advice